api/go.mod

```diff
@@ -4,30 +4,31 @@ go 1.20
 
 require (
 	github.com/google/gofuzz v1.2.0
-	github.com/metal3-io/ip-address-manager/api v1.4.1
+	github.com/metal3-io/ip-address-manager/api v1.4.1-0.20230801074326-623678ce1ae7
```

Comment:
I would suggest bumping IPAM to CAPI v1.5.0 first, release IPAM, come back here and bump IPAM to that new version for easier dependency cycle

Reply:
We already uplifted IPAM and specifying uplifted commit here, since there was not a release on ipam yet